docs/feature-store/sources-targets.md

```diff
@@ -59,13 +59,6 @@ df = fs.ingest(
 
 ## Kafka source
 
-
-```{admonition} Note
```

Comment:
is it supported as tech preview. PLease return these lines but delete the extra heading